doc/how_to_guides/parallel_planning/src/parallel_planning_main.cpp

```diff
@@ -0,0 +1,300 @@
+class Demo
+{
+public:
+  /// @brief
+  /// @param node
+  Demo(rclcpp::Node::SharedPtr node)
+    : node_{ node }
+    , moveit_cpp_{ std::make_shared<moveit_cpp::MoveItCpp>(node) }
+    , planning_component_{ std::make_shared<moveit_cpp::PlanningComponent>(PLANNING_GROUP, moveit_cpp_) }
+    , visual_tools_(node, "panda_link0", "parallel_planning_example", moveit_cpp_->getPlanningSceneMonitor())
+  {
+    moveit_cpp_->getPlanningSceneMonitor()->providePlanningSceneService();
+
+    visual_tools_.deleteAllMarkers();
+    visual_tools_.loadRemoteControl();
+
+    Eigen::Isometry3d text_pose = Eigen::Isometry3d::Identity();
+    text_pose.translation().z() = 1.75;
+    visual_tools_.publishText(text_pose, "Parallel Planning Tutorial", rvt::WHITE, rvt::XLARGE);
+    visual_tools_.trigger();
+  }
+
+  bool loadPlanningSceneAndQuery()
+  {
+    std::string hostname = "";
+    int port = 0.0;
+    std::string scene_name = "";
+
+    node_->get_parameter_or(std::string("warehouse.host"), hostname, std::string("127.0.0.1"));
+    node_->get_parameter_or(std::string("warehouse.port"), port, 33829);
+
+    if (!node_->get_parameter("warehouse.scene_name", scene_name))
+    {
+      RCLCPP_WARN(LOGGER, "Warehouse scene_name NOT specified");
+    }
+
+    moveit_warehouse::PlanningSceneStorage* planning_scene_storage = nullptr;
+
+    // Init DB connection
+    try
+    {
+      warehouse_ros::DatabaseLoader db_loader(node_);
+      warehouse_ros::DatabaseConnection::Ptr warehouse_connection = db_loader.loadDatabase();
+      warehouse_connection->setParams(hostname, port, 20);
+      if (warehouse_connection->connect())
+      {
+        planning_scene_storage = new moveit_warehouse::PlanningSceneStorage(warehouse_connection);
+        RCLCPP_INFO(LOGGER, "Connected to DB: '%s'", hostname.c_str());
+      }
+      else
+      {
+        RCLCPP_ERROR(LOGGER, "Failed to connect to DB");
+        return false;
+      }
+    }
+    catch (std::exception& e)
+    {
+      RCLCPP_ERROR(LOGGER, "Failed to initialize planning scene storage: '%s'", e.what());
+      return false;
+    }
+
+    // Load planning scene
+    moveit_msgs::msg::PlanningScene scene_msg;
+    try
+    {
+      if (!planning_scene_storage)
+      {
+        RCLCPP_ERROR(LOGGER, "No planning scene storage");
+        return false;
+      }
+
+      if (planning_scene_storage->hasPlanningScene(scene_name))  // Just the world (no robot)
+      {
+        moveit_msgs::msg::PlanningSceneWorld world_meta_data;
+        if (!planning_scene_storage->getPlanningSceneWorld(world_meta_data, scene_name))
+        {
+          RCLCPP_ERROR(LOGGER, "Failed to load planning scene world '%s'", scene_name.c_str());
+          return false;
+        }
+        scene_msg.world = world_meta_data;
+        scene_msg.robot_model_name =
+            "NO ROBOT INFORMATION. ONLY WORLD GEOMETRY";  // this will be fixed when running benchmark
+      }
```

Comment:
This looks very "debuggy". Did you intend to have this in? 

If so, maybe do not make it all caps and make the second sentence a little more descriptive, for example

"No robot information. Using only world geometry."